TWLight/emails/tasks.py

```diff
@@ -192,15 +206,39 @@ def send_survey_active_user_emails(sender, **kwargs):
         base=base_url, id=survey_id, lang=survey_lang
     )
 
-    email = SurveyActiveUser()
+    template_email = SurveyActiveUser()
 
-    email.send(
+    email_message = template_email.make_email_object(
         user_email,
         {
             "lang": user_lang,
             "link": link,
         },
+        connection=connection,
+    )
+    # Save as a djmail instance
+    model_instance = Message.from_email_message(email_message)
+    if hasattr(email_message, "priority"):
+        if email_message.priority <= Message.PRIORITY_LOW:
+            model_instance.priority = email_message.priority
+            model_instance.status = Message.STATUS_PENDING
+    model_instance.created_at = timezone.now()
+    model_instance.uuid = uuid4()
+    model_instance.save()
+
+    # send email and update state in djmail
+    _safe_send_message(model_instance, connection)
```

Comment:
Obviously, this is an internal method. Forking djmail would be cleaner, but this works for now.